server/src/test/java/io/spine/server/storage/RecordStorageTest.java

```diff
@@ -162,12 +161,9 @@ void filterByColumns() {
         EntityRecord notFineRecord1 = buildStorageRecord(idWrong1, newState(idWrong1));
         EntityRecord notFineRecord2 = buildStorageRecord(idWrong2, newState(idWrong2));
 
-        EntityRecordWithColumns recordRight =
-                create(fineRecord, matchingEntity, storage);
-        EntityRecordWithColumns recordWrong1 =
-                create(notFineRecord1, wrongEntity1, storage);
-        EntityRecordWithColumns recordWrong2 =
-                create(notFineRecord2, wrongEntity2, storage);
+        EntityRecordWithColumns recordRight = create(fineRecord, matchingEntity);
```

Comment:
I will be merging with this forever. You have asked me to modify this in 2.x branch already.

Please rollback.

Reply:
As discussed vocally, this change was needed to address `BadImport` warning on usage of statically imported `EntityRecordWithColumns.create()`.